test/test-call-forwarding

```diff
@@ -171,4 +196,8 @@ def main(args):
 
 if __name__ == "__main__":
 	args = parse_arguments()
-	main(args)
+
+	try:
+		main(args)
+	except KeyboardInterrupt as e:
```

Comment:
Handling the interrupt is nice, but I think that adding some timeout (~30 secs) to exit the script would be good to have too.